luigi/worker.py

```diff
@@ -238,8 +238,11 @@ class DequeQueue(collections.deque):
     deque wrapper implementing the Queue interface.
     """
 
-    put = collections.deque.append
-    get = collections.deque.pop
+    def put(self, *args, **kwargs):
```

Comment:
The point is to ignore 'timeout=N' that some parts of the code include in the call.